debian/control

```diff
@@ -10,8 +10,10 @@ Homepage: https://learningequality.org/kolibri
 
 Package: kolibri-server
 Architecture: all
-Depends: kolibri (>= 0.12.6), nginx-full, uwsgi (>= 2.0.12), uwsgi-plugin-python3, redis-server (>=4.0)
 Recommends: anacron
+Depends: kolibri (>= 0.12.6), nginx-full, uwsgi (>= 2.0.12), uwsgi-plugin-python3, redis-server (>=4.0), python3 (>= 3.4)
+Pre-Depends: bash
```

Comment:
bash is an `essential` package. No need to add this dependency.
```
apt-cache show bash
Package: bash
Architecture: amd64
Version: 4.4.18-2ubuntu1.2
Multi-Arch: foreign
Priority: required
Essential: yes

```

Reply:
Removed it again, I thought you had mentioned it more literally as a dependency earlier